src/convolution.cpp

```diff
@@ -382,6 +379,39 @@ ConvolutionDescriptor::ForwardGetValidWorkSpaceSizeGemm(Handle& handle,
 #endif
 }
 
+std::size_t
+ConvolutionDescriptor::BackwardGetValidWorkSpaceSizeGemm(const TensorDescriptor& dyDesc,
```

Comment:
This function exist prior this PR. I just moved it during refactoring in commit (05) 46dc7c2b150ebe23d6f8aeac8d4030aa9d71f7e0. AFAICS these functions use shortened direction naming, like `Forward`, `Backward` and `WrW` instead of `ForwardData`, `BackwardData`, `BackwardWeights`. It is expected that these functions will be removed when GEMM solvers are ready.

Thanks for reviewing!